projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -1521,6 +1523,26 @@ public function get_author( $author, $show_email_and_ip = false ) {
 			$author['site_visible'] = $site_visible;
 		}
 
+		// Only include WordPress.com user data when author_wpcom_data is enabled.
+		$args              = $this->query_args();
+		$author_wpcom_data = isset( $args['author_wpcom_data'] ) && $args['author_wpcom_data'];
+
+		if ( ! empty( $id ) && $author_wpcom_data ) {
```

Comment:
What do you think about simplifying it into this:

```suggestion
		$args = $this->query_args();

		if ( ! empty( $id ) && ! empty( $args['author_wpcom_data'] ) ) {
```